docs/_writing_dependencies_conda.rst

```diff
@@ -0,0 +1,318 @@
+        ---------------------------|-----------------
+        seqtk-1.2                  |                0          29 KB  bioconda
+
+    The following NEW packages will be INSTALLED:
+
+        seqtk: 1.2-0   bioconda
+        zlib:  1.2.8-3
+
+    Fetching packages ...
+    seqtk-1.2-0.ta 100% |#############################################################| Time: 0:00:00 444.71 kB/s
+    Extracting packages ...
+    [      COMPLETE      ]|################################################################################| 100%
+    Linking packages ...
+    [      COMPLETE      ]|################################################################################| 100%
+    #
+    # To activate this environment, use:
+    # > source activate __seqtk@1.2
+    #
+    # To deactivate this environment, use:
+    # > source deactivate __seqtk@1.2
+    #
+    $ which seqtk
+    seqtk not found
+    $
+
+The above install worked properly, but seqtk is not on your ``PATH`` because this merely
+created an environment within the Conda directory for the seqtk installation. Planemo
+will configure Galaxy to exploit this installation. If you wish to interactively explore
+the resulting enviornment to explore the installed tool or produce test data the output
+of the ``conda_env`` command can be sourced.
+
+::
+
+    $ . <(planemo conda_env seqtk_seq.xml)
+    Deactivate environment with conda_env_deactivate
+    (seqtk_seq) $ which seqtk
+    /home/planemo/miniconda2/envs/jobdepsiJClEUfecc6d406196737781ff4456ec60975c137e04884e4f4b05dc68192f7cec4656/bin/seqtk
+    (seqtk_seq) $ seqtk seq
+
+    Usage:   seqtk seq [options] <in.fq>|<in.fa>
+
+    Options: -q INT    mask bases with quality lower than INT [0]
+             -X INT    mask bases with quality higher than INT [255]
+             -n CHAR   masked bases converted to CHAR; 0 for lowercase [0]
+             -l INT    number of residues per line; 0 for 2^32-1 [0]
+             -Q INT    quality shift: ASCII-INT gives base quality [33]
+             -s INT    random seed (effective with -f) [11]
+             -f FLOAT  sample FLOAT fraction of sequences [1]
+             -M FILE   mask regions in BED or name list FILE [null]
+             -L INT    drop sequences with length shorter than INT [0]
+             -c        mask complement region (effective with -M)
+             -r        reverse complement
+             -A        force FASTA output (discard quality)
+             -C        drop comments at the header lines
+             -N        drop sequences containing ambiguous bases
+             -1        output the 2n-1 reads only
+             -2        output the 2n reads only
+             -V        shift quality by '(-Q) - 33'
+             -U        convert all bases to uppercases
+             -S        strip of white spaces in sequences
+    (seqtk_seq) $ conda_env_deactivate
+    $
+
+As shown above the ``conda_env_deactivate`` will be created in this environment and can
```

Comment:
Why not use `deactivate` that should be present in that conda environment? Less of a change for people that know conda outside of the galaxy/planemo universe.